src/store/actions/relatedLinks.actions.js

```diff
@@ -29,8 +52,8 @@ export default {
     if (!getters.isLoggedIn) {
       Vue.toasted.error('You must login to remove your link',{
         singleton: true,
-        theme: "bubble",
-        position: "bottom-center",
+        theme: 'bubble',
+        position: 'bottom-center',
         duration : 700
```

Comment:
This seems fast. I noticed this timeout elsewhere. Wouldn't it be interesting to keep it longer? It looks hard to read

Reply:
Sure, this is just the current standard I've seen throughout the codebase. Do you have a suggested duration? I can make these 3 seconds, that's usually a good average time for users, especially since these messages tend to be on the shorter side.

Reply:
I'm good with 3s. I don't think we need to be too concerned about the exact time. 3 is fine!